docs/source/stats.rst

```diff
@@ -494,3 +494,25 @@ to verify in an observational setting.
 
    Mediation
    MediationResults
+
+
+Oaxaca-Blinder Decomposition
+----------------------------
+
+This package attempts to port the functionality of the oaxaca command in STATA to Python. 
```

Comment:
One final one.  Could remove `package` here, and probably move this to the end of the paragraph resphasing to something like:

This function has been tested against the ``oaxaca`` comment in Stata.